pipeline-runner/R/qc-1-filter_emptydrops.R

```diff
@@ -98,7 +96,7 @@ filter_emptydrops <- function(scdata, config, sample_id, task_name = 'classifier
   # get filter stats after filtering
   filter_stats <- list(
     before = before,
```

Comment:
take a look and see if there is a good reason to calculate before a bit differently than the other functions.

Reply:
changed to be like others